euphonic/sampling.py

```diff
@@ -0,0 +1,352 @@
+"""Functions for sampling properties in high-dimensional space.
+
+These are implemented in a generic way for application in euphonic.powder
+"""
+
+from itertools import product
+import numpy as np
+from typing import Iterator, Tuple
```

Comment:
PEP8 imports:

```
from itertools import product
from typing import Iterator, Tuple

import numpy as np
```

I know not every Euphonic module sticks to this, but this is a chance to start fixing that...